app/views/home/index.html.haml

```diff
@@ -1,2 +1,20 @@
-#page-content-wrapper.w-100
-  %h1.text-center Welcome to Book Swap App
+#page-content-wrapper.w-100.h-100
+  %h1.text-center.mb-5 Welcome to Book Swap App
+
+  = render 'form'
+
+  .mt-5.border.rounded-lg.shadow-sm
```

Comment:
This looks very similar to the table from `books/index.html.haml`. Could we maybe extract this in a partial, which we use both in `books/index` and `home/index`?

Reply:
Since these tables are a bit different, I decided not to extract them to partial yet.